manifests/kustomize/base/installs/multi-user/pipelines-profile-controller/sync.py

```diff
@@ -74,62 +84,53 @@ def get_settings_from_env(controller_port=None,
         disable_istio_sidecar if disable_istio_sidecar is not None \
             else os.environ.get("DISABLE_ISTIO_SIDECAR") == "true"
 
-    settings["minio_access_key"] = \
```

Comment:
Why are you removing this section ? We want to keep the minio secrets and use them for seaweedfs